pytorch/buildspec.yml

```diff
@@ -118,62 +118,62 @@ images:
     context:
       <<: *TRAINING_CONTEXT
   # BuildEC2CPUPTInferencePy3DockerImage:
-    # <<: *INFERENCE_REPOSITORY
```

Comment:
I see that all the inference DLCs are commented. Is that intended?

Reply:
it was that way before and it's only the position of the hashes that are different. I tested the inference images and then reverted back as this is for the wheel exception and not to enable inference builds.